types/error-cause/index.d.ts

```diff
@@ -0,0 +1,17 @@
+// Type definitions for error-cause 1.0
+// Project: https://github.com/es-shims/error-cause
+// Definitions by: Conrad Buck <https://github.com/conartist6>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+type ErrorTypes = [
+    'Error',
+    'AggregateError',
+    'EvalError',
+    'RangeError',
+    'ReferenceError',
+    'SyntaxError',
+    'TypeError',
+    'URIError',
+];
```

Comment:
```suggestion
] as const;
```

Reply:
I don't think that compiles. It's a tuple type not an array literal. An array literal can't exist in a `d.ts` file.

Reply:
ohh right, you're correct. my bad